manual/installation.tex

```diff
@@ -292,13 +318,14 @@ \subsection{Role of QMC\_DATA}
 QMC_DATA/NiO/NiO-fcc-supertwist111-supershift000-S128.h5
 QMC_DATA/NiO/NiO-fcc-supertwist111-supershift000-S256.h5
 QMC_DATA/NiO/NiO_afm_fcidump.h5
-QMC_DATA/NiO/NiO_afm_wfn.dat 
-QMC_DATA/NiO/NiO_nm_choldump.h5   
+QMC_DATA/NiO/NiO_afm_wfn.dat
+QMC_DATA/NiO/NiO_nm_choldump.h5
 \end{verbatim}
-
+\normalsize
```

Comment:
I sometimes see normalsize, sometimse don't. Could you explain why?

Reply:
You shouldn't see it at all.  Will fix.